non_modular_snn/snn_model.py

```diff
@@ -697,13 +706,15 @@ def set_mode(mode, modes):
                         help='Number of excitatory output neurons. The number of inhibitory neurons are defined the same.')
     parser.add_argument('--threshold_i', type=int, default=0, 
                         help='Threshold value used to ignore the hyperactive neurons.')
+    parser.add_argument('--seed', type=int, default=0, 
+                        help='Set seed for random generator.')
 
     parser.add_argument('--ad_path_test', type=str, default="_test_E{}", 
                         help='Additional string arguments to use for saving test outputs in testing')
-    parser.add_argument('--ad_path', type=str, default="_offset{}")             
+    parser.add_argument('--ad_path', type=str, default="_offset{}_S{}")             
     parser.add_argument('--multi_path', type=str, default="epoch{}_T{}_T{}") 
 
-    parser.add_argument('--process_mode', type=str, choices=["train", "test"], default="test", 
+    parser.add_argument('--process_mode', type=str, choices=["train", "record", "calibrate", "test"], default="test", 
```

Comment:
Record and calibrate are non-standard terminology and should be explained, also not contained in the help string currently.

Reply:
Added in the help string and relevant README files